unpacker.json

```diff
@@ -11450,5 +11450,53 @@
 			"expectedmin": 0.0,
 			"expectedmax": 1
 		}
+	},
+	"0x504":{ 
+		"DV_ACU_SYSTEM_STATUS":{
+			"type": "unit",
```

Comment:
Should this type be `Uint`?

Reply:
![image](https://user-images.githubusercontent.com/59113973/115899695-5a5fac80-a45f-11eb-853b-6dc3067d8cc8.png)

I got a syntax error here, as well